### DISCUSSION TITLE
Is dark mode for the localhost gui?

### DISCUSSION THREAD

**Discussion by m-GDEV** [ActivityWatch/activitywatch#548] [Q&A] [answered]
title.

not sure if this is being worked on, but i think this would be beneficial for everyone.

**Comment by ErikBjare** [comment, marked as answer]
There's a feature request for this on the forum: https://forum.activitywatch.net/t/dark-theme-for-web-ui-and-android-app/274

However, we're not making it a priority, since ActivityWatch works decently with extensions such as [Dark Reader](https://darkreader.org/).

**Reply by m-GDEV** [reply]
ok cool, thank you.
